packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -359,6 +364,7 @@ class VideoPlayerOptions {
   VideoPlayerOptions({
     this.mixWithOthers = false,
     this.allowBackgroundPlayback = false,
+    this.webOptions,
```

Comment:
Should the web options be added to the `VideoPlayerOptions` object, or instead just used by the new `setWebOptions` method?